terraform/environments/mlra/application_variables.json

```diff
@@ -9,27 +9,29 @@
       "instance_type": "m5.large",
       "key_name": "",
       "ecr_url": "374269020027.dkr.ecr.eu-west-2.amazonaws.com/mlra-ecr-repo",
```

Comment:
Not for now, but as a reminder for ourselves we need to get rid of these account numbers and reference them from the data outputs that MP provide.